src/styles/blocks/card.scss

```diff
@@ -0,0 +1,120 @@
+.card {
+  width: 200px;
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  background-color: $white-color;
+  border: 1px solid $gray-elements-color;
+  border-radius: 5px;
+
+  @include content-inline-padding;
+
+  &__image {
```

Comment:
merge rules with `&__img`. Remove redundant styles. And check for other selectors I noticed at html